kernel/traffic/core/src/main/java/org/apache/shardingsphere/traffic/algorithm/traffic/segment/SQLMatchTrafficAlgorithm.java

```diff
@@ -42,7 +43,9 @@ public final class SQLMatchTrafficAlgorithm implements SegmentTrafficAlgorithm {
     @Override
     public void init(final Properties props) {
         Preconditions.checkArgument(props.containsKey(SQL_PROPS_KEY), "%s cannot be null.", SQL_PROPS_KEY);
-        sql = getExactlySQL(props.getProperty(SQL_PROPS_KEY));
+        String sqlValue = props.getProperty(SQL_PROPS_KEY);
+        Preconditions.checkArgument(!(StringUtils.isEmpty(sqlValue) || sqlValue.length() == 0), "%s cannot be empty.", SQL_PROPS_KEY);
```

Comment:
Use `ShardingSpherePreconditions` to replace `Preconditions`, and then define an accurate exception class, refer to #24939

Reply:
I have made some changes, please check.